superflore/utils.py

```diff
@@ -198,6 +200,10 @@ def get_license(l):
         return 'Boost-1.0'
     elif re.search(pub_dom_re, l, f):
         return 'public_domain'
+    elif re.search(government_re, l, f):
+        return 'GPR'
+    elif re.search(proprietary_re, l, f):
+        return 'Proprietary'
```

Comment:
So I don't think we can pull this change in. Gentoo doesn't play nice with these strings. While we can certainly add the "GPR" license, the "Proprietary" string is rather meaningless.

At one point in time, I had an idea to package the license for any "Proprietary" package by fetching it from the repo, but this was rather low on the TODO-list for superflore. 

I don't know what Open Embedded has for this situation, but, from time to time, we have to add licenses to the overlay to support installation of certain packages. They go to [this folder](https://github.com/ros/ros-overlay/tree/master/licenses).

Maybe I'll add that feature to superflore soon (it would capture the LICENSE file from the repo, then it would add the new license to the overlay in the PR for the update).

Reply:
Upon looking at this package further, I found that it lacks a LICENSE file completely. I would file an issue on that repo and state that the repo lacks a LICENSE file, and should have one (so that we can clearly see the terms that are agreed upon by accepting the GPR license).

Reply:
I think that doing something where if it doesn't match any of our pre-categorized licenses it would be classified as "Custom" or something catchall. And capturing the raw license file from the repo would be valuable.

There's a level that we're just trying to group the licenses for easier review. If we can't classify it we should just pass it through unclassified.

Reply:
Ptal at the modified patch that passes the licenses through unclassified when we can't recognize and categorize. I believe we'll not have a use for UnknownLicense exception if we'll pass it through when unrecognized, so please let me know if I should remove it.

Submitted https://github.com/swri-robotics/qt_metapackages/issues/1 asking for adding licensing files there.